### PR TITLE
Followers and followings list

### PR DESCRIPTION
**Pull request by shiara02**
### OVERVIEW

-Followers and followings list on profile page
-User can follow or unfollow the users on the list

### NOTES

-For now, only the user can see his followers and followings list because there isn't a profile view yet.

### SNIPPETS

image
image
image

### CODE REVIEW

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

File: src/app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -5,20 +5,28 @@
...
+        $this->userFollowService = $userFollowService;
+    }
+
     public function show(): View
     {
-        $user = optional(User::find(auth()->id()));
+        $user = User::find(auth()->id());
 
         if (!$user) {
             return view('user.profile', ['posts' => collect()]);
```
Comment (hipe-francis) [inline src/app/Http/Controllers/UserController.php:25]: Do you really need to add this condition here? Seems kinda pointless if the allowed user to access this route is always authenticated.

**Review by hipe-francis — APPROVED** [review]
GEFM 🚀

File: src/app/Http/Controllers/UserController.php (hunk 1 above)
Comment (shiara02) [reply src/app/Http/Controllers/UserController.php:25]: Upon checking on CI po sir, it gives me an error if i don't put that condition. But I asked sir raf on that matter na po, I will edit po once it is confirmed if okay lang po i-ignore yung sa CI. Thank you po!

**Pull request merged**